internal/commands/user/flags.go

```diff
@@ -9,22 +16,31 @@ const (
 	flagPendingShort = "p"
 	flagPendingUsage = `select to show users with pending status`
 
+	flagStatus      = "status"
+	flagStatusShort = "p"
+	flagStatusUsage = `select the state of users to fiilter by, available options: ["enabled", "disabled"]`
+
 	flagProvider      = "provider"
 	flagProviderShort = "t"
-	flagProviderUsage = `set the provider types for which to filter the list of app users with, available options: ["local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token"]`
+	flagProviderUsage = `set the provider types for which to filter the list of app users with, available options: ` +
+		`["local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "oauth2-apple", ` +
+		`"anon-user", "custom-token", "custom-function"]`
 
-	flagUser      = "user"
-	flagUserShort = "u"
-	flagUserUsage = `set the user ids for which to filter the list of app users with`
+	flagUser            = "user"
+	flagUserShort       = "u"
+	flagUserUsage       = `set the user ids for which to filter the list of app users with`
+	flagUserDeleteUsage = `set the user ids for which to delete from the app`
 )
 
 const (
-	providerTypeLocalUserPass = "local-userpass"
-	providerTypeAPIKey        = "api-key"
-	providerTypeFacebook      = "oauth2-facebook"
-	providerTypeGoogle        = "oauth2-google"
-	providerTypeAnonymous     = "anon-user"
-	providerTypeCustom        = "custom-token"
+	providerTypeLocalUserPass  = "local-userpass"
+	providerTypeAPIKey         = "api-key"
+	providerTypeFacebook       = "oauth2-facebook"
+	providerTypeGoogle         = "oauth2-google"
+	providerTypeAnonymous      = "anon-user"
+	providerTypeCustom         = "custom-token"
```

Comment:
[question] did you get this `Custom` name from somewhere?  wondering if we want it to be `CustomToken` to avoid any ambiguity with `CustomFunction`.  what do you think?